lib/logux/policy_caller.rb

```diff
@@ -2,26 +2,39 @@
 
 module Logux
   class PolicyCaller
-    attr_reader :action, :meta, :policy
+    attr_reader :action, :meta, :policy, :stream
 
-    def initialize(action:, meta:)
+    def initialize(action:, meta:, stream:)
```

Comment:
pass stream to the policy is not the good idea, it will be better to handle `Logux::NoPolicyError` in controller and do all the logging/streaming there, @wilddima wdyt?

Reply:
@NikitaNaumenko are you read for the final fix to merge this PR :)

Reply:
@ai Yes, i read it, and fix it soon. But now i have no idea how i get kind of event raise NoPolicyError exception, or i what action raise NoActionError.  Are you have idea what solution will be better?)